frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -5,18 +5,22 @@ action.GitMachete.CheckoutSelectedBranchAction.text=Checkout Selected Branch
 action.GitMachete.CheckoutSelectedBranchAction.GitMacheteContextMenu.text=_Checkout Branch
 action.GitMachete.CheckoutSelectedBranchAction.description=Checkout this branch
 
-action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Rebase Selected Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=_Rebase Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Rebase this branch onto parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Checkout and Rebase Selected Branch Onto Parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=Checkout and _Rebase Branch Onto Parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Checkout and rebase this branch onto parent
 
 action.GitMachete.PullSelectedBranchAction.text=Pull Selected Branch
-action.GitMachete.PullSelectedBranchAction.GitMacheteContextMenu.text=_Pull Branch
+action.GitMachete.PullSelectedBranchAction.GitMacheteContextMenu.text=Pull Branch
 action.GitMachete.PullSelectedBranchAction.description=Pull this branch
 
 action.GitMachete.PushSelectedBranchAction.text=Push Selected Branch...
 action.GitMachete.PushSelectedBranchAction.GitMacheteContextMenu.text=_Push Branch...
 action.GitMachete.PushSelectedBranchAction.description=Push this branch using push dialog
 
+action.GitMachete.ResetSelectedBranchAction.text=Checkout and Reset Selected Branch to Remote
+action.GitMachete.ResetSelectedBranchAction.GitMacheteContextMenu.text=Checkout and Reset Branch to Remote
```

Comment:
What about marking some shortcut key here? Maybe `s`(oft)?
`Checkout and Re_set Branch to Remote`